commune_app/all_models/users.py

```diff
@@ -37,11 +37,11 @@ def save(self, *args, **kwargs):
         self.full_clean()
         return super().save(*args, **kwargs)
 
-    def vote_on_decision(decision_id):
-        pass
-
-    def add_decision(args):
-        pass
-
-    def execute_chore():
-        pass
+    def join_commune(self, commune_id):
+        if (self.commune_id is not None and self.commune_id != commune_id):
+            raise Exception("already partner in another commune")
```

Comment:
maybe you can check the name of the commune and present it here
it could be more user friendly

Reply:
I think it's more convenient to transfer an id because anyway you put the id in the appropriate field and you don't have to get confused with his name, like this:
`def join_commune(self, commune_id):
        if self.commune_id != commune_id:
            raise Exception("already partner in another commune")
        self.commune_id = commune_id`

Reply:
In case the user uses the id you are right.
but in this case, I think that the user uses the commune name and in the backend, we filter by the name and find the commune_id so if I"ll write an error with the commune_id it will be useless for the user.

Reply:
ok

Reply:
i hope i understand you
 check my commits